packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -738,7 +744,10 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
       labelWidth += Math.ceil(longestLabelWidth);
     }
 
-    let minChartWidth = this.margins.left! + this.margins.right! + labelWidth * (this._tickValues.length - 1);
+    let minChartWidth =
```

Comment:
Could you explain what this issue is

Reply:
The issue was due to the changes of PR: https://github.com/microsoft/fluentui/pull/28239. Changes done in the above PR are dependent on the first render cycle computations. So after merging it with my changes for optimization of first render cycle the chart was not rendering due to NPE issues at various places.

Reply:
Got it.